docs/notes-base/platforms.md

```diff
@@ -85,12 +86,13 @@ Android deployment is possible from Windows, Linux and Mac OSX.
 The Android engine supports devices using ARMv7 or ARMv8 processors.
```

Comment:
THis is out of date too isn't it?

We now support x86, x86-64, ARM and ARM64

Reply:
Oops :)

PR updated!